src/fxmaster.js

```diff
@@ -151,7 +151,7 @@ Hooks.on("dropCanvasData", async (canvas, data) => {
     flags: {},
     height: data.height,
     hidden: false,
-    img: data.file,
+    texture: {src: data.file},
```

Comment:
This change is required otherwise lint will complain
```suggestion
    texture: { src: data.file },
```